contracts/staking/GoodFundManager.sol

```diff
@@ -290,66 +298,62 @@ contract GoodFundManager is DAOUpgradeableContract, DSMath {
 	/**
 	 * @dev  Function that get addresses of staking contracts that we can collect interest from with a specific gas limit
 	 * but that also pass the interest>=gascosts*multiplier
-	 * @param _maxGasAmount The maximum amount of the gas that keeper willing to spend collect interests
-	 * @return addresses of the staking contracts to the collect interests
+	 * @return addresses of the staking contracts to the collect interests , interests of the staking contracts ,
+	 * collected interest amounts until the contract including it , gas spent amounts until the contract including it,max gas amount can spend on the interestcollect transaction
+	 * if actual spend gas equal or smaller than max gas amount
```

Comment:
need to correct description + make it more clear, shorter lines

Reply:
Fixed it